src/plugins/specload/specload.c

```diff
@@ -565,7 +565,7 @@ int isChangeAllowed (Key * oldKey, Key * newKey)
 		return 0;
 	}
 
-	keyswitch_t changes = keyCompare (oldKey, newKey);
+	int changes = keyCompare (oldKey, newKey);
```

Comment:
Why not `elektraKeyFlags`?
```suggestion
	elektraKeyFlags changes = keyCompare (oldKey, newKey);
```